src/containers/app/AppSagas.js

```diff
@@ -171,12 +183,20 @@ function* switchOrganizationWatcher() :Generator<*, *, *> {
 
 function* switchOrganizationWorker(action :Object) :Generator<*, *, *> {
 
-  // TODO: this was added in quickly, might need refactoring
-  AccountUtils.storeOrganizationId(action.orgId);
-
-  // not ideal since it resets clears form inputs, but none of that is being stored in redux at the moment anyway...
-  // TODO: check action.orgId !== selectedOrganizationId
-  yield put(push(Routes.HOME_PATH));
+  try {
+    const { value } = action;
```

Comment:
Isn't it `action.orgId`?

Reply:
Now that `selectOrganization` is a proper `RequestSequence` instead of a custom redux action creator, it is no longer `action.orgId` and follows the traditional `SequenceAction` shape.